templates/primary-secondary/primary-secondary.js

```diff
@@ -1078,7 +1076,12 @@ class TemplatePrimarySecondary extends LocalizeCoreElement(LitElement) {
 			</div>
 		`;
 
-		if (this.hasForm) return html`<d2l-form hide-error-summary @d2l-form-invalid=${this.#handleInvalidForm} @d2l-form-submit=${this.#handleFormSubmit} @d2l-form-dirty=${this.#handleFormDirty}>
+		if (this.hasForm) return html`<d2l-form
+			hide-error-summary
```

Comment:
Could form just always hide its error summary if it's provided with a `summary-id`? It's kind of like saying "don't render your own summary, I've got my own and here's its id".